docs/deployment/k8s.md

```diff
@@ -397,6 +397,19 @@ backend:
       password: ${POSTGRES_PASSWORD}
 ```
 
+Please also change the host and port environment variables in the `app-config.production.yaml` to the following:
+
+```yaml
```

Comment:
This is the same as above. Thinking it makes sense to just add a notice about remembering to apply this to `app-config.production.yaml` as well if you have one?

Reply:
Makes more sense. I'll do that. Thanks